frontend/src/pages/admin/FlowBlueprints/index.vue

```diff
@@ -150,7 +181,33 @@ export default {
                 }
             })
             this.teamTypes.sort((A, B) => { return A.order - B.order })
+        },
+        exportFlowBlueprints () {
+            FlowBlueprintsApi.exportFlowBlueprints()
+                .then((data) => {
+                    const filename = `blueprints_export_${dateToSlug(new Date())}.json`
+                    downloadData(data.blueprints, filename)
+                })
+                .catch(err => console.error(err.message))
+        },
+        onImportFlowBlueprints (blueprints) {
+            FlowBlueprintsApi.importFlowBlueprints(blueprints)
+                .then(response => this.setBlueprints(response.blueprints))
+                .then(() => {
+                    Alerts.emit('Blueprints successfully imported!', 'confirmation', 7500)
+                })
+                .catch(err => {
+                    Alerts.emit('Something went wrong!', 'warning', 7500)
```

Comment:
_Just some thoughts..._

I am always uneasy / unsure what we should be putting in these alerts. 

Yes, it is good to feedback to the user but it doesnt tell them anything like "check you file" etc

Reply:
I couldn't find a good and easy way to check the validity of the imported file/text and wanted to let the user know that his action didn't follow through. I could definitely come back with an improvement on validation